src/lang/it.js

```diff
@@ -4,7 +4,8 @@ const it = {
     contact: 'Contatti',
     experience: 'Esperienza professionale',
     education: 'Formazione',
-    skills: 'Competenze'
+    skills: 'Competenze',
+    about: 'A proposito di me'
```

Comment:
Not sure about "a proposito di me", even if it's correct it sounds a bit weird.
I'd go for something like "Chi sono" that could be literally translated in "Who I am".